doxygen/main.md

```diff
@@ -40,7 +40,8 @@ DMX receiver / RDM responder.
 
 ## Common {#main-features-common}
 
-- \ref usb-logging, with adjustable log levels.
+- \ref usb-logging, with adjustable log levels. Requires Linux or MAC OS X 10.7 or
```

Comment:
Re-ordering to list Mac or Linux would make it even clearer that the 10.7 requirement doesn't apply to Linux.